src/pages/instances/forms/UploadExternalFormatFileForm.tsx

```diff
@@ -0,0 +1,334 @@
+      }
+
+      // If the image is already in raw format, remove the format conversion option
+      // this will optimise the conversion process since raw vm images do not need to be converted
+      const isRawImage = await isImageTypeRaw(file);
+      if (isRawImage) {
+        await formik.setFieldValue("formatConversion", false);
+      }
+    }
+  };
+
+  const handleCloseModal = useCallback(() => {
+    formik.resetForm();
+    close();
+    notify.clear();
+  }, [formik.resetForm, close, notify]);
+
+  const noFileSelectedMessage = !formik.values.imageFile
+    ? "A file must be selected first."
+    : "";
+
+  return (
+    <>
+      <ScrollableContainer
+        className={classnames({
+          "fixed-height": fixedFormHeight,
+          "u-hide": uploadState,
+        })}
+        dependencies={[]}
+        belowIds={["modal-footer"]}
+      >
+        <Form onSubmit={formik.handleSubmit}>
+          <Input
+            id="image-file"
+            name="imageFile"
+            type="file"
+            label="External format (.qcow2, .vmdk, etc...)"
+            labelClassName="u-hide"
+            onChange={(e) => void handleFileChange(e)}
+          />
+          <Input
+            {...formik.getFieldProps("name")}
+            id="name"
+            type="text"
+            label="New instance name"
+            placeholder="Enter name"
+            error={formik.touched.name ? formik.errors.name : null}
+            disabled={!!noFileSelectedMessage}
+            title={noFileSelectedMessage}
+          />
+          <StoragePoolSelector
+            project={project?.name || ""}
+            value={formik.values.pool}
+            setValue={(value) => void formik.setFieldValue("pool", value)}
+            selectProps={{
+              id: "pool",
+              label: "Root storage pool",
+              disabled: !project || !!noFileSelectedMessage,
+              title: noFileSelectedMessage,
+            }}
+          />
+          <Select
+            {...formik.getFieldProps("architecture")}
+            id="architecture"
+            label="Image architecture"
+            options={archOptions}
+            disabled={!!noFileSelectedMessage}
+            title={noFileSelectedMessage}
+          />
+          <label htmlFor="">Conversion options</label>
+          <Input
+            {...formik.getFieldProps("formatConversion")}
+            type="checkbox"
+            label={
+              <>
+                Convert to raw format{" "}
+                <Icon
+                  name="information"
+                  title={
+                    noFileSelectedMessage
+                      ? noFileSelectedMessage
+                      : "Can be skipped if the image is already in raw format to speed up the import."
+                  }
+                />
+              </>
+            }
+            disabled={!!noFileSelectedMessage}
+            checked={formik.values.formatConversion}
```

Comment:
```suggestion
            checked={formik.values.formatConversion}
            title={noFileSelectedMessage}
```
We should have the title on the checkboxes as well.

Reply:
for some reason when the input is a checkbox, `title` has no effect if specified on the `Input` component directly. Instead I added `title` attribute to the label.

Reply:
I think this is because the checkbox gets wrapped by the label.